MoviesMVVM/MoviesMVVM/WebViewController.swift

```diff
@@ -52,19 +51,26 @@ final class WebViewController: UIViewController {
 
     private func loadWebViewData() {
```

Comment:
```suggestion
    private func loadWebViewData() {
```
в вьюмодель сервисы и запрос